core/API/Request.php

```diff
@@ -412,6 +412,9 @@ public static function getRequestParametersGET()
             return array();
         }
         $GET = UrlHelper::getArrayFromQueryString($_SERVER['QUERY_STRING']);
+        foreach ($GET as $name => $value) {
+            $GET[$name] = urldecode($value);
+        }
```

Comment:
API requests use output sanitization, so shouldn't be an issue there.
Unless I am mistaken controllers don't use rendered API responses.
So if a visualization doesn't do output sanitization, there will be an issue, though that would affect more than URL decoding.

Could do this only for getRawSegmentFromRequest, though this weird no URL decoding needs to be fixed eventually.